backend/src/main/java/gov/cdc/usds/simplereport/service/OrganizationService.java

```diff
@@ -54,6 +59,9 @@ public OrganizationService(OrganizationRepository repo,
                 .collect(Collectors.toList());
         List<Organization> validOrgs = _repo.findAllByExternalId(candidateExternalIds);
         if (validOrgs == null || validOrgs.size() != 1) {
+            Integer numOrgs = (validOrgs == null) ? 0
```

Comment:
Note: can use `int` instead of `Integer`. Also I think that the repository always returns a non-null list, but meh, null checks are not expensive.